libs/wallet-ui/src/hooks/use-explorer-url.tsx

```diff
@@ -9,3 +9,23 @@ export const useExplorerUrl = () => {
 
   return config?.apps.explorer
 }
+
+export type ExplorerLinks = {
+  txUrl: (hash: string | null | undefined) => string | undefined
+  partyUrl: (key: string | null | undefined) => string | undefined
+  blockUrl: (number: string | null | undefined) => string | undefined
+}
+export const useExplorerLinks = (): ExplorerLinks => {
+  const explorerUrl = useExplorerUrl()
+  const base = explorerUrl?.trim().replace(/\/*$/, '')
+  const canCreateLink = (value: string | null | undefined) =>
+    base && value && value.length > 0
+  return {
+    txUrl: (hash: string | null | undefined) =>
```

Comment:
suggestion (non-blocking): i think it would be a lot more intuitive to read if these props were called `getTxUrl`, `getPartyUrl`, `getBlockUrl`. otherwise i'd assume they are not functions but strings